src/main/java/au/org/emii/ncdfgenerator/NcdfEncoder.java

```diff
@@ -96,22 +101,20 @@ public InputStream get() throws Exception
 		try {
 			if( featureInstancesRS.next())
 			{
-				// munge
 				long instanceId = -1234;
 				Object o = featureInstancesRS.getObject(1);
-				Class clazz = o.getClass();
-				if( clazz.equals( Integer.class )) {
+
+				// munge
+				if( o instanceof Integer)
```

Comment:
From the Retrieving Values from Result Sets section of the Java tutorials:

JDBC allows a lot of latitude as far as which getXXX methods you can use to retrieve the different SQL types. For example, the method getInt can be used to retrieve any of the numeric or character types. The data it retrieves will be converted to an int; that is, if the SQL type is VARCHAR , JDBC will attempt to parse an integer out of the VARCHAR. The method getInt is recommended for retrieving only SQL INTEGER types, however, and it cannot be used for the SQL types BINARY, VARBINARY, LONGVARBINARY, DATE , TIME, or TIMESTAMP.